### DISCUSSION TITLE
unable to load any VST3 plugins in Windows 10 / VS2017

### DISCUSSION THREAD

**Discussion by gtf180** [apohl79/audiogridder#668] [Q&A]
I use vs2017 to compile and generate audiogridderinst.vst3, and use JuseHost to load then JuseHost say: "scan complete. Note that the following files appeared to be plugin files, but failed to load correctly: audiogridderinst.vst3". 
  but AudioGridderServer.exe is working. any plugin(AudioGridder.vst3、AudioGridderInst.vst3、AudioGridderMidi.vst3) can not working by VS2017. even I try loading the VST3 in REAPER,it`s still。but I use Plugin Installer is working。Please tell me the possible problems

**Comment by apohl79** [comment]
No idea what your problem is, I don't understand it from your description tbh. I am using VS2019 on windows.
